### PR TITLE
Adapt native tooltip for new structure of NcAvatar

### LINKED ISSUE #36357: [BITV]: Adapt tooltips on NcAvatar for files_sharing
## Problem

Custom tooltips were already exchanged by native tooltips. Though some small structure should be adapted. Now the usage of `aria-label` and `title` is not needed anymore. It is possible just to use `display-name` attribute as before.

## Solution

Remove `title` and `aria-label` attributes from NcAvatar components in files_sharing.

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
- [x] Blocked by https://github.com/nextcloud/nextcloud-vue/issues/3677

Resolves: https://github.com/nextcloud/server/issues/36357

## Summary

Remove title and aria-label attributes from NcAvatar components in files_sharing.

|  Before   | After                                                        |
|-------------|---------------------------------------------------------------|
| Screenshot from 2023-01-25 15-26-14 | Screenshot from 2023-01-25 15-22-51 | 
| Screenshot from 2023-01-25 15-25-58 |  Screenshot from 2023-01-25 15-23-03 |

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Comment by nickvergessen** [comment]
Conflicting files
dist/files_sharing-files_sharing_tab.js
dist/files_sharing-files_sharing_tab.js.map

File: apps/files_sharing/src/components/SharingEntry.vue [hunk 1]
```diff
@@ -25,7 +25,8 @@
 		<NcAvatar class="sharing-entry__avatar"
 			:is-no-user="share.type !== SHARE_TYPES.SHARE_TYPE_USER"
 			:user="share.shareWith"
-			:title="share.type === SHARE_TYPES.SHARE_TYPE_USER ? share.shareWithDisplayName : ''"
+			:display-name="share.shareWithDisplayName"
+			:tooltip-message="share.type === SHARE_TYPES.SHARE_TYPE_USER ? share.shareWith : ''"
```
Comment (nickvergessen) [inline apps/files_sharing/src/components/SharingEntry.vue:29]: `tooltip-message` does not exist. Also you are fixing the lib at the moment, so using the title property should be fine?

Also `share.shareWith` is not good to use in the UI. It's the UID not the display name, so something like `0e7a88c6-9cc9-11ed-a8fc-0242ac120002` for certain LDAP setups, etc.

Comment (JuliaKirschenheuter) [reply apps/files_sharing/src/components/SharingEntry.vue:29]: Thanks Joas! i haven't seen that `tooltip-message` were added from previous versions, of coarse i will take it away. Actually i want to have only :display-name="share.shareWithDisplayName". For now i will set this PR into draft.

**Comment by JuliaKirschenheuter** [comment]
can't do `/backport to stable25` yet because stable25 has "@nextcloud/vue": "^7.5.0" https://github.com/nextcloud/server/blob/stable25/package.json#L49

**Comment by Pytal** [comment]
> can't do `/backport to stable25` yet because stable25 has "@nextcloud/vue": "^7.5.0" [`stable25`/package.json#L49](https://github.com/nextcloud/server/blob/stable25/package.json?rgh-link-date=2023-02-24T15%3A28%3A30Z#L49). Needed version: [nextcloud/nextcloud-vue@`v7.6.0` (release)](https://github.com/nextcloud/nextcloud-vue/releases/tag/v7.6.0)

- [ ] https://github.com/nextcloud/server/pull/36841

**Review by Pytal — APPROVED** [review]

File: apps/files_sharing/src/components/SharingEntry.vue (hunk 1 above)
Comment (artonge) [reply apps/files_sharing/src/components/SharingEntry.vue:29]: Is this really solved ?

Comment (JuliaKirschenheuter) [reply apps/files_sharing/src/components/SharingEntry.vue]: Thnaks @artonge, now is solved. I've left only `:display-name="share.shareWithDisplayName"`

**Review by artonge — APPROVED** [review]

**Pull request merged**